### PR TITLE
Event gallery Added for Issue#6

### PR DESCRIPTION
**Pull request by nikhil3000**
Gallery feature has been added. I have added an index.js in the root directory which will pull data from google drive folder (set id of target folder in line 6 of index.js). It will then create gallery.json which will be further used to display images event wise as shown in the image attached

image

image

Event name will be same as that of subfolder name as shown in google drive screenshot.
image

P.S My UI is too bad, so please take care of UI

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -20,10 +21,27 @@ router.get('/', function(req, res, next) {
...
+        {
+            break;
+        }
+    }
+    
+    res.render('pages/eventGallery',{title: data[i].title, Images:data[i].images});
+
+});
+
+router.get('/talks/:id', function(req, res, next) {
```
Comment (anistark) [inline server/routes/index.js]: Why this?

File: server/views/pages/eventGallery.html [hunk 2]
```diff
@@ -0,0 +1,232 @@
+{% extends '../layout.html' %} {% block title %}{% endblock %} {% block content %}
```
Comment (anistark) [inline server/views/pages/eventGallery.html]: View file names should be in snake case. NOT camel case.

File: server/views/partials/head_includes.html [hunk 3]
```diff
@@ -4,3 +4,5 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/prefixfree/1.0.7/prefixfree.min.js"></script>
 <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
 <script src="https://use.fontawesome.com/5fecaaf616.js"></script>
+<script type="text/javascript" src="/client/assets/gallery.json"></script>
```
Comment (anistark) [inline server/views/partials/head_includes.html]: This should only be included on the gallery page. not globally.

File: token.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (anistark) [inline token.json]: What's the purpose of this file?

Comment (nikhil3000) [reply token.json]: This is a temporary token created by google drive API.

P.S Thanks for the detailed analysis. I'll make the necessary changes.

**Pull request merged**